alacritty.yml

```diff
@@ -488,6 +488,7 @@
   #   binding:
   #     key: U
   #     mods: Control|Shift
+  #     mode: None
```

Comment:
Adding a new mode would require adding it to the documentation in the bindings section, but I'm not sure it's a good idea to add `None` as a mode, since there is no "None" mode.

This is a list of modes and notmodes, so `~None` would be a thing too, which makes no sense really. If anything the default would just be an empty string.

Reply:
Yeah I added the `none` mode because I couldn't figure out a good way to show the new field in the configuration file while also having both of the `ModeWrapper`'s `BindingMode` members remain empty. That's why I only wrote `none` in instead of both `none` and `~none,` since it would purely mean to return the default `ModeWrapper` in its deserialize implementation. I can change this if you like though and just simply omit the field from the binding example in config and remove the `None` mode if you prefer.

Reply:
Just `mode: ""` would work. But since mode is somewhat of an "advanced" feature I'd be kinda okay with just leaving it out too tbh.

Reply:
Alright sounds good to me.

Reply:
fixed.